util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -196,9 +196,10 @@ def sanitize_method(method: Optional[str]) -> Optional[str]:
     method = method.upper()
     if (environ.get(OTEL_PYTHON_INSTRUMENTATION_HTTP_CAPTURE_ALL_METHODS) or
         # Based on https://www.rfc-editor.org/rfc/rfc7231#section-4.1 and https://www.rfc-editor.org/rfc/rfc5789#section-2.
+        # Defined list: https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/http/http-spans.md#common-attributes
         method in ["GET", "HEAD", "POST", "PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE", "PATCH"]):
         return method
-    return "UNKNOWN"
+    return "_OTHER"
```

Comment:
we also need to populate `http.request.method_original` but only on spans

Reply:
This will require some refactoring. Is this new to the HTTP sem conv, or should we do this regardless of the stability mode?

Reply:
Confirmed this is only for new stable HTTP